Misc/NEWS.d/next/Core and Builtins/2018-10-15-20-41-18.bpo-34990.NVDjdP.rst

```diff
@@ -0,0 +1 @@
+Use an unsigned long for the timestamp for the .pyc file in compileall.py
```

Comment:
You describe the fix, but you should describe the fixed bug instead: compileall now supports timestamps newer than year 2038.

Reply:
+1